schema/schema.sql

```diff
@@ -0,0 +1,63 @@
+CREATE DATABASE catalog;
+
+CREATE TABLE items (
+  id BIGINT PRIMARY KEY,
+  genre_id BIGINT NOT NULL,
+  author_id BIGINT NOT NULL,
+  label_id BIGINT NOT NULL,
+  publish_date DATE NOT NULL,
+  archived BOOLEAN NOT NULL,
+  FOREIGN KEY (genre_id) REFERENCES genres(id),
+  FOREIGN KEY (author_id) REFERENCES authors(id),
+  FOREIGN KEY (label_id) REFERENCES labels(id)
+);
```

Comment:
- [ ] In SQL, there is no way to represent inheritance. Therefore, I would advise you to not create the Items table but create the tables that inherit from the Item class (`Books`, `MusicAlbums`, `Games`). Kindly remove the Items table. 👍